BuiltInSpicyAnalyzer.cmake

```diff
@@ -52,6 +52,11 @@ function (spicy_add_analyzer)
             ${CMAKE_CURRENT_BINARY_DIR}/${NAME_LOWER}_spicy_init.cc
             ${CMAKE_CURRENT_BINARY_DIR}/${NAME_LOWER}_spicy_hooks_${SPICY_ANALYZER_NAME}.cc)
 
+        # CXX files given to SOURCES are added to the lib target
+        # separately from generated_sources.
+        set(cxx_sources ${SPICY_ANALYZER_SOURCES})
+        list(FILTER cxx_sources INCLUDE REGEX ".*\.(c|cc|cxx|cpp|c\\+\\+)$")
```

Comment:
Idea, feel free to drop: since this is intended for internal use we could require `cc` here.

Reply:
Ah, good call.

I'll keep just `.*\.cc$` then as we still need to filter out `.spicy` and `.evt` files.

Thanks!